### ISSUE TITLE
Print image and boxes with highest loss

### ISSUE THREAD

**Issue by holger-prause** [AlexeyAB/darknet#6337]
Hello,
During training i sometimes have a very high loss for certain images / classes / boxes and i see this is on the console:
(Note this happens - after a lot of batches where map is already somewhat ok - so its not after starting training where this is expected)
I.E

`v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 139 Avg (IOU: 0.687133, GIOU: 0.677602), Class: 0.803532, Obj: 0.423531, No Obj: 0.000725, .5R: 0.793651, .75R: 0.523810, count: 63, class_loss = 23.561205, iou_loss = 735.645447, total_loss = 759.206665`

**I want to find out which image / class / box produced this output**, so i can adopt my labeling strategy.My object are very diverse and i am not sure if my mindset aka model is consistent enough, the objects bounding boxes are properly labeled but maybe i need to split into seperate classes as the network fails to generalize or my model is inconsistent.
Heres my questions:

**1) Is it technical possible to retrieve these information?** 
I want to find the training sample and if possible the labeled box which produces the highest loss.
I am aware that we train in batched and maybe its just not possible or very complicated to determine the training sample for this.
Its also clear to me that the network makes suggestions and you measure how good it matches the ground truth, so you just cannot tell which class/box exactly produced this "problem". But at least the batch with training samples should be clear, if not the image itsef. The rest you would need to find out by looking at the activation maps i guess(I already asked how to do this but got no answer - which is fine).

1.1) If yes - is this build in or could you point me to a code line where i could implement this myself?
Thank you very much, i hope i can get an aswer, if not its ok. The yolo code is available so you can always look for yourself.
But a little push in the right direction would help a lot.

Greetings,
Holger
